### PR TITLE
Add VOLTOCONFIG Env Var

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

**Comment by stevepiercy** [comment]
@sneridagh there is [another open PR for environment variables documentation](https://github.com/plone/volto/pull/5736) that needs some review and love before the docs from this one can be included.

File: docs/source/configuration/environmentvariables.md [hunk 1]
```diff
@@ -175,6 +175,15 @@ You can also generate builds on your continuous integration, then deploy them an
     yarn
     BUILD_DIR=dist node dist/server.js
     ```
+
+`VOLTOCONFIG`
+    You can also provide a custom location for `volto.config.js` via an environment variable.
```
Comment (stevepiercy) [inline docs/source/configuration/environmentvariables.md:180]:
```suggestion
    This environment variable allows you to specify a custom location for {file}`volto.config.js`.
```

File: docs/source/configuration/environmentvariables.md [hunk 2]
```diff
@@ -175,6 +175,15 @@ You can also generate builds on your continuous integration, then deploy them an
...
+
+    It can be relative to the current project or absolute.
+
+    ```shell
+     VOLTOCONFIG=../../volto.config.js yarn start
```
Comment (stevepiercy) [inline docs/source/configuration/environmentvariables.md:185]:
```suggestion
    VOLTOCONFIG=../../volto.config.js yarn start
```

File: docs/source/configuration/volto-config-js.md [hunk 3]
```diff
@@ -62,3 +62,23 @@ module.exports = {
   theme
 };
 ```
+
+## Using `VOLTOCONFIG` environment variable
```
Comment (stevepiercy) [inline docs/source/configuration/volto-config-js.md:66]:
```suggestion
## `VOLTOCONFIG` environment variable usage
```

File: docs/source/configuration/volto-config-js.md [hunk 4]
```diff
@@ -62,3 +62,23 @@ module.exports = {
...
+
+You can also provide a custom location for `volto.config.js` via an environment variable.
```
Comment (stevepiercy) [inline docs/source/configuration/volto-config-js.md:68]:
```suggestion
This environment variable allows you to specify a custom location for {file}`volto.config.js`.
```

File: docs/source/configuration/volto-config-js.md [hunk 5]
```diff
@@ -62,3 +62,23 @@ module.exports = {
...
+
+```shell
+VOLTOCONFIG=../../volto.config.js yarn start
+```
+
+```shell
+VOLTOCONFIG=$(pwd)/volto.config.js yarn start
+```
+
+you can also do it in core (monorepo):
```
Comment (stevepiercy) [inline docs/source/configuration/volto-config-js.md:80]: I think you mean this, but maybe not. I'm not really sure of your intent.

```suggestion
You can also set it from the root of the monorepo:
```

File: packages/registry/news/5752.feature [hunk 6]
```diff
@@ -0,0 +1 @@
+Add VOLTOCONFIG environment variable @sneridagh
```
Comment (stevepiercy) [inline packages/registry/news/5752.feature:1]:
```suggestion
Add `VOLTOCONFIG` environment variable. @sneridagh
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Minor tweaks and MyST seasoning.

**Comment by stevepiercy** [comment]
Outside the scope of this issue, of course, but copy-paste in docs always leads to drift and more maintenance work. I wonder if we should audit the [Configuration](https://6.docs.plone.org/volto/configuration/index.html) section and reorganize it? I can create an issue for future work, and add it to the GSoD draft proposal.

**Comment by sneridagh** [comment]
@stevepiercy I thought on it already, but it's so small, and it won't change...

**Pull request merged**